lib/api/objectDelete.js

```diff
@@ -54,8 +54,9 @@ export default function objectDelete(authInfo, request, log, cb) {
                     contentLength: objMD['content-length'],
                 });
             }
-            return services.deleteObject(bucketName, objMD, objectKey, log,
-                err => {
+
+            return services.deleteObject(bucketName, objMD, objectKey,
```

Comment:
this looks like we are just changing a line break.  let's revert so that no changes appear in this file.